src/core/trulens/core/utils/imports.py

```diff
@@ -42,19 +42,20 @@ def safe_importlib_package_name(package_name: str) -> str:
     )
 
 
-def requirements_of_file(path: Path) -> Dict[str, requirements.Requirement]:
+def requirements_of_file(path: str) -> Dict[str, requirements.Requirement]:
     """Get a dictionary of package names to requirements from a requirements
     file."""
+    _trulens_resources = resources.files("trulens.core")
```

Comment:
Behaviour changed but documentation doesn't mention the restriction to `trulens.core`.

Reply:
oh this was because it was broken at the time that I wrote this but it seems to be fixed in main now so I just reverted it.

Reply:
I take it back, I remember what was wrong now after one of my tests failed. I've renamed the function to mention it's about trulens core (it's only used for that purpose).

In case you want to know why I had to change it, we have to zip the trulens.core package for it to run in Snowflake because it's not in the internal conda channel. However, when Snowflake reads the zip file it doesn't actually decompress it, it just somehow reads it in place. The way this `imports.py` file was working before it read the requirements file but because it was still in the compressed zip it actually first uncompressed it into some temp file. But this temp file immediately died after `static_resources` returned.